src/devices/Mhz19b/samples/Mhz19b.Sample.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.IO.Ports;
+using System.Text;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Mhz19b.Samples
+{
+    /// <summary>
+    /// Sample for MH-Z19B sensor
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Main entry point
+        /// </summary>
+        public static void Main(string[] args)
+        {
+            // create serial port using the setting acc. to datasheet, pg. 7, sec. general settings
+            var serialPort = new SerialPort("/dev/serial0", 9600, Parity.None, 8, StopBits.One);
+            serialPort.Encoding = Encoding.ASCII;
+            serialPort.ReadTimeout = 1000;
+            serialPort.WriteTimeout = 1000;
+            serialPort.Open();
+            Mhz19b sensor = new Mhz19b(serialPort.BaseStream, true);
+
+            // Alternatively you can let the binding create the serial port stream:
+            // Mhz19b sensor = new Mhz19b("/dev/serial0");
+
+            // Switch ABM on (default).
+            // sensor.SetAutomaticBaselineCorrection(AbmState.On);
+
+            // Set sensor detection range to 2000ppm (default).
+            // sensor.SetSensorDetectionRange(DetectionRange.Range2000);
+
+            // Perform calibration
+            // Step #1: perform zero point calibration
+            // Step #2: perform span point calibration at 2000ppm
+            // CAUTION: enable the following lines only if you know exactly what you do.
+            //          Consider also that zero point and span point calibration are performed
+            //          at different concentrations. The sensor requires up to 20 min to be
+            //          saturated at the target level.
+            // sensor.PerformZeroPointCalibration();
+            // ---- Now change to target concentration for span point.
+            // sensor.PerformSpanPointCalibration(VolumeConcentration.FromPartsPerMillion(2000));
+
+            // Continously read current concentration
+            while (true)
+            {
+                try
+                {
+                    VolumeConcentration reading = sensor.GetCo2Reading();
+                    Console.WriteLine($"{reading}");
```

Comment:
Similar as for Temperature and  Ratio, do something like reading.unityouwanttodisplay for clarity (where unityouwanttodisplay is the proper unit)

Reply:
Same comment as for the Ahtxx sample.

Reply:
Done